go.mod

```diff
@@ -4,7 +4,8 @@ go 1.18
 
 require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
-	github.com/cosmos/cosmos-sdk v0.45.5
+	github.com/cosmos/cosmos-sdk v0.46.1
```

Comment:
Not sure we can switch to 46 here

Reply:
oh, it just jumps when I do go mod tidy, will change it back to 0.45.5

Reply:
Ah I see. I'll be upgrading this to 0.45.8 soon, which should fix this